backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -327,6 +327,7 @@ public RepositoryEntity getRepositoryById(Session session, RepositoryIdentificat
             LOGGER);
         repository.update(request);
       }
+      repository.setDeleted(true);
```

Comment:
we are again holding the transaction open till the uac operation is completed. 
can we close the transaction post the saveOrUpdate on next line.

Reply:
Fixed